packages/services/api/src/modules/schema/module.graphql.ts

```diff
@@ -296,13 +296,22 @@ export default gql`
   type SchemaChange {
     criticality: CriticalityLevel!
     criticalityReason: String
-    message: String!
+    message(
+      """
+      Whether to include a note about the safety of the change based on usage data within the message.
+      """
+      withSafeBasedOnUsageNote: Boolean = true
```

Comment:
We have legacy clients (CLI) that don't select the `isSafeBasedOnUsage` field.
Those should still receive the same message as before.